app/javascript/packages/verify-flow/index.tsx

```diff
@@ -16,9 +16,18 @@ interface VerifyFlowProps {
    * The path to which the current step is appended to create the current step URL.
    */
   basePath: string;
+
+  /**
+   * URL to which user should be redirected after completing the form.
+   */
+  completionURL: string;
 }
 
-export function VerifyFlow({ initialValues = {}, basePath }: VerifyFlowProps) {
+export function VerifyFlow({ initialValues = {}, basePath, completionURL }: VerifyFlowProps) {
+  function onComplete() {
```

Comment:
Would we want to pass in an `onComplete` so we could spy on it? Or do we just stub `window.location` in tests make sure it works

Reply:
Hm, yeah, I could see it working either way, but I like that idea to push out all the window-related handling to pack, which would be easier to stub the package behavior, since [stubbing location can be pretty gross](https://github.com/18F/identity-idp/blob/87dfefd2e2e508fffc84ebedc0643a0f4917d6a2/spec/javascripts/packs/form-steps-wait-spec.js#L200-L210).

Reply:
Updated in rebased [f74f2c0](https://github.com/18F/identity-idp/pull/6229/commits/f74f2c0b6f466b2deb4e78cbd8dd6a4b570c7fa3).